package/ios/RNCSliderComponentView.mm

```diff
@@ -77,7 +77,14 @@ - (void)tapHandler:(UITapGestureRecognizer *)gesture {
 
     CGPoint touchPoint = [gesture locationInView:slider];
     float rangeWidth = slider.maximumValue - slider.minimumValue;
-    float sliderPercent = touchPoint.x / slider.bounds.size.width;
+    
+    float sliderPercent;
+    if ([UIView userInterfaceLayoutDirectionForSemanticContentAttribute:slider.semanticContentAttribute] == UIUserInterfaceLayoutDirectionRightToLeft) {
```

Comment:
Can't/Shouldn't we use `inverted` prop here?

Reply:
I'm not passing the `inverted` prop, it's automatically gets inverted on the RTL layout. so in this case `inverted` would be `false`.